assets/src/dashboard/components/contentGutter/index.js

```diff
@@ -17,18 +17,18 @@
 /**
  * External dependencies
  */
-import styled from 'styled-components';
+import styled, { css } from 'styled-components';
 
 export const StandardViewContentGutter = styled.div(
-  ({ theme }) => `
-    margin: 0 ${theme.DEPRECATED_THEME.standardViewContentGutter.desktop}px;
+  ({ theme }) => css`
+    margin: 0 52px;
 
-    @media ${theme.DEPRECATED_THEME.breakpoint.largeDisplayPhone} {
-      margin: 0 ${theme.DEPRECATED_THEME.standardViewContentGutter.largeDisplayPhone}px;
+    @media ${theme.breakpoint.mobile} {
+      margin: 0 10px;
     }
 
-    @media ${theme.DEPRECATED_THEME.breakpoint.smallDisplayPhone} {
-      margin: 0 ${theme.DEPRECATED_THEME.standardViewContentGutter.smallDisplayPhone}px;
+    @media ${theme.breakpoint.min} {
+      margin: 0 10px;
```

Comment:
Don't have to do this now since it's code cleanup that's kinda unrelated to the intent of this PR, but I don't think we have to worry about any breakpoints in the dashboard beneath the min-width you added to the content area since the content will never shrink beneath that

Reply:
i think this clean up will go nicely with the work to update the grid since that factors in the gutter and fun math.